languages/csharp/exercises/concept/constructors/.docs/after.md

```diff
@@ -0,0 +1,73 @@
+Creating an instance of a _class_ is done by calling its [_constructor_][constructors] through the [`new` operator][new]. A constructor is a special type of method whose goal is to initialize a newly created instance. [Constructors look like regular methods][constructor-syntax], but without a return type and with a name that matches the classes' name.
+
+```csharp
+class Library
+{
+    private books;
+
+    public Library()
+    {
+        this.books = 10;
+    }
+}
+
+// This will call the constructor
+var library = new Library();
+```
+
+Like regular methods, constructors can have parameters. Constructor parameters are usually stored as (private) [fields][fields] to be accessed later, or else used in some off-time calculation. Arguments can be passed to constructors just like passing arguments to regular methods.
+
+```csharp
+class Building
+{
+    private int numberOfStories;
+    private int totalHeight;
+
+    public Building(int numberOfStories, double storyHeight)
+    {
+        this.numberOfStories = numberOfStories;
+        this.totalHeight = numberOfStories * storyHeight;
+    }
+}
+
+// Call a constructor with two arguments
+var largeBuilding = new Building(55, 6.2)
+```
+
+Specifying a constructor is optional. If no constructor is specified, a [parameterless constructor][parameterless-constructors] is generated by the compiler:
```

Comment:
Is it worth calling out the generated constructor is public?

Reply:
I considered this, but I decided not to mention this now as I want to mention this when discussing private constructors. Would that make sense?

Reply:
That makes sense to me